lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -148,6 +160,15 @@ def __init__(self, namespace: str):
         self.client = Client()
         self.namespace = namespace
 
+    def delete_pod(self, pod_name: str) -> None:
+        """Deleting given pod.
+
+        Args:
+            pod_name (str): Pod name
+
+        """
+        self.client.delete(Pod, pod_name, namespace=self.namespace)
+
```

Comment:
Should this change be part of this PR?

Reply:
This belongs to another PR, it should be removed.